examples/gno.land/r/demo/boards2/board.gno

```diff
@@ -96,21 +97,37 @@ func (board *Board) DeleteThread(pid PostID) {
 	}
 }
 
-func (board *Board) Render() string {
-	s := "\\[" + newLink("post", board.GetPostFormURL()) + "]\n\n"
-	if board.threads.Size() > 0 {
-		board.threads.Iterate("", "", func(_ string, v interface{}) bool {
-			post := v.(*Post)
-			if post.isHidden {
-				return false
-			}
+// Render renders a board into Markdown.
+//
+// Pager is used for pagination if it's not nil.
+func (board *Board) Render(p *PaginationOpts) string {
```

Comment:
Maybe, it might be possible to use a struct instead of a pointer everywhere when using `PaginationOpts` as argument:

```suggestion
func (board *Board) Render(p PaginationOpts) string {
```

Reply:
@jeronimoalbi no, it isn't as this is optional for nested renders in threads and posts. Pointer kept in board to use the same func as for threads and posts.